bot.py

```
@@ -156,3 +156,51 @@ async def shutdown() -> None:
+@app_commands.describe(item="The emoji or item you want to test on the cat (e.g. 🥒, 🐟, 🧶)")
+async def hisstest(interaction: discord.Interaction, item: str):
+    # Clean up whitespace from input
+    clean_item = item.strip()
+
+    # Check if the offered item exists in our special list
+    if clean_item in CAT_REACTIONS:
+        reaction = CAT_REACTIONS[clean_item]
+        response_text = f"**You offered:** {clean_item}\n**Cat's Reaction:** {reaction['emoji']} {reaction['text']}"
+    else:
+        # Pick a random "not interested" response for any other emoji
+        random_response = random.choice(UNINTERESTED_RESPONSES)
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)